src/Infrastructure/WB.Infrastructure.Native/Storage/Postgre/Implementation/PostgresKeyValueStorageWithCache.cs

```diff
@@ -25,7 +26,7 @@ public override TEntity GetById(string id)
         {
             return memoryCache.GetOrCreate(CachePrefix + id, cache =>
             {
-                lock (CachePrefix)
+                lock (lockObj)

```

Comment:
this will be lock on any operation with differant entities. Old lock works only for one entity. Need return old code.

Reply:
Generic instances will be created per type parameter. Every instance will have it's own lock.

Reply:
make static lockObj